x-pack/legacy/plugins/code/server/routes/file.ts

```diff
@@ -18,15 +18,17 @@ import { TEXT_FILE_LIMIT } from '../../common/file';
 import { decodeRevisionString } from '../../common/uri_util';
 
 export function fileRoute(server: CodeServerRouter, gitOps: GitOperations) {
-  async function repoExists(req: hapi.Request, repoUri: string) {
+  async function getRepoUriFromMeta(
```

Comment:
nit: since you rename it to `getRepoUriFromMeta`, it might make more sense to just return a `string | undefined`, so that when it's `undefined`, the repo does not exsit.

Reply:
Understood. I was trying to avoid expressing the validity of the returned value with undefined, but it actually didn't seem to be clear enough. I have made the change as you suggested, please review it again.